advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -3,12 +3,41 @@ title: Installing EDB Wait States
 navTitle: Installing
 ---
 
-## On Linux 
+## On all Linux systems except Debian and Ubuntu
 
-To install EDB Wait States on a Linux system:
+To install EDB Wait States on any Linux operating system except Debian or Ubuntu:
 
-1.  Install EDB Wait States with the `edb-asxx-server-edb_wait_states` RPM package, where `xx` is the EDB Postgres Advanced Server version number.
+1.  Enter the following command:
 
+  ```shell
+  sudo <package-manager> -y install <install-command>
```

Comment:
Install command doesn't sound like the correct word to use here. Maybe `install-package-name` or something like that. This isn't really a command.

Reply:
Revised to use "install-package-name"